tests/integration/report.py

```diff
@@ -109,6 +83,9 @@ def pytest_runtest_logreport(self, report):
             self.test_data[report.nodeid] = outcome
 
     def pytest_sessionfinish(self, session):
+        # disabled
```

Comment:
Could we get more context in the comment on why this is disabled? It seems temporary since no code was removed. When do we anticipate it being re-enabled?

Reply:
yeah I was just trying to get the new options to work and all this reporting code (which has bit rotten a bunch and makes too many assumptions) was getting in the way. I will re-enable it soon.

Reply:
undid this now